internal/app/apid/pkg/director/director.go

```diff
@@ -62,6 +63,11 @@ func (r *Router) Director(ctx context.Context, fullMethodName string) (proxy.Mod
 
 	switch {
 	case okNodes:
+		// COSI methods do not support one-2-many proxying.
+		if strings.HasPrefix(fullMethodName, "/cosi.") {
```

Comment:
this is a safety belt - one2many proxy will corrupt protobufs of COSI APIs, so better to have a better explicit error